src/components/spec-document/try-it/TryItAuth.vue

```diff
@@ -15,42 +16,121 @@
-          :id="`auth-token-tooltip-${getSchemeLabel(scheme)}-${data.id}`"
-          :text="scheme.description"
-        />
-      </InputLabel>
-      <input
-        :id="`auth-token-input-${getSchemeLabel(scheme)}-${data.id}`"
-        v-model="tokenValueMap[key]"
-        :aria-describedby="`auth-token-tooltip-${getSchemeLabel(scheme)}-${data.id}`"
-        autocomplete="off"
-        placeholder="App credential"
-        type="text"
+      <div
+        v-if="scheme.type === 'http' && scheme.scheme === 'basic'"
       >
+        <div class="param-wrapper">
+          <InputLabel
+            class="param-label"
+            :for="`auth-token-input-basic-username-${data.id}`"
+          >
+            Username
+            <Tooltip
+              v-if="scheme.description"
+              :id="`auth-token-tooltip-basic-username-${data.id}`"
+              :text="scheme.description"
+            />
+          </InputLabel>
+          <input
+            :id="`auth-token-input-basic-username-${data.id}`"
+            v-model="authInputs[`${key}-username`]"
+            :aria-describedby="`auth-token-tooltip-basic-username-${data.id}`"
+            autocomplete="off"
+            placeholder="Enter Username"
+            type="text"
+          >
+        </div>
+        <div class="param-wrapper">
+          <InputLabel
+            class="param-label"
+            :for="`auth-token-input-basic-password-${data.id}`"
+          >
+            Password
+            <Tooltip
+              v-if="scheme.description"
+              :id="`auth-token-tooltip-basic-password-${data.id}`"
+              :text="scheme.description"
+            />
+          </InputLabel>
+          <input
+            :id="`auth-token-input-basic-password-${data.id}`"
+            v-model="authInputs[`${key}-password`]"
+            :aria-describedby="`auth-token-tooltip-basic-password-${data.id}`"
+            autocomplete="off"
+            placeholder="Enter Password"
+            type="password"
+          >
+        </div>
+      </div>
+      <div
+        v-else-if="scheme.type === 'http' && scheme.scheme === 'bearer'"
+      >
+        <div class="param-wrapper">
+          <InputLabel
+            class="param-label"
+            :for="`auth-token-input-bearer-token-${data.id}`"
+          >
+            JWT token
+            <Tooltip
+              v-if="scheme.description"
+              :id="`auth-token-tooltip-bearer-token-${data.id}`"
+              :text="scheme.description"
+            />
+          </InputLabel>
+          <input
+            :id="`auth-token-input-bearer-token-${data.id}`"
+            v-model="authInputs[`${key}-token`]"
+            :aria-describedby="`auth-token-tooltip-bearer-token-${data.id}`"
+            autocomplete="off"
+            placeholder="Enter JWT token"
+            type="password"
+          >
+        </div>
+      </div>
+      <div
+        v-else
+      >
+        <div class="param-wrapper">
```

Comment:
nite: we can collapse the parent div (with conditionals) and the wrapper div, for all three label-input pairs in this component

```suggestion
        <div v-else class="param-wrapper">
```

Reply:
param-wrapper wraps each label/input pair, ,outer div wraps entire schema block (when there are multiple inputs. Yes, we can do it for blocks with single input, but I wanted it to be consistent and easy add stuff inside of this wrapper.